apps/console/src/features/applications/pages/application-edit.tsx

```diff
@@ -98,6 +100,17 @@ const ApplicationEditPage: FunctionComponent<ApplicationEditPageInterface> = (
     const [ isApplicationRequestLoading, setApplicationRequestLoading ] = useState<boolean>(false);
     const [ inboundProtocolList, setInboundProtocolList ] = useState<string[]>(undefined);
     const [ inboundProtocolConfigs, setInboundProtocolConfigs ] = useState<Record<string, any>>(undefined);
+    const [ isDescTruncated, setIsDescTruncated ] = useState<boolean>(false);
+
+    useEffect(() => {
+        if (appDescElement) {
+            const nativeElement = appDescElement.current;
+
+            if (nativeElement && (nativeElement.offsetWidth < nativeElement.scrollWidth)) {
+                setIsDescTruncated(true);
+            }
+        }
+    }, [ appDescElement, isApplicationRequestLoading ]);
```

Comment:
Do we need this here? `isApplicationRequestLoading`

Reply:
Yes @thivi. Even though `useRef` calls twice, the `PageLayout` component doesn't render the passed children immediately (it will use a placeholder loading component), when that happens the relative element always returns `0` as the offset height and widths. So, I'm relying on this boolean variable to re-render it for the third time so it returns correct values for figuring out whether the element's content is truncated or not.

Reply:
Resolved with [16fa93d](https://github.com/wso2/identity-apps/pull/3028/commits/16fa93d6d651893ee81b2b86acad68a9a0ffe23c)